assets/data-port/import/upload-level/index.test.js

```diff
@@ -0,0 +1,172 @@
+import { render } from '@testing-library/react';
+import { UploadLevels } from './index';
+
+describe( '<UploadLevels />', () => {
+	it( 'should not be ready when no file is uploaded', async () => {
```

Comment:
I believe these tests can be sync, right?

Reply:
Oops! This is a leftover from when I was trying to fix the test. I'll remove it in my next PR.